### ISSUE TITLE
Connect: Generate shared process protos the same way as tshd protos

### ISSUE THREAD

**Issue by ravicious** [gravitational/teleport#15955]
This issue is blocked until we move webapps into the teleport repo.

The shared process protos are generated using [a completely standalone process](https://github.com/gravitational/webapps/tree/179390265c482f188d374e9470488257dc0d8e1e/packages/teleterm#generating-shared-process-grpc-protobuf-files). This means they're not linted and formatted.

We should make it so that they're generated in the same way as the tshd protos in lib/teleterm.

**Comment by zmb3** [comment]
@ravicious is this work complete?

**Comment by ravicious** [comment]
No, it's about the gRPC server/client we use for PTY processes. https://github.com/gravitational/teleport/pull/21452#discussion_r1100039089

But this is super low priority work, we haven't modified those protos ever since Grzegorz implemented the shared process for that one Electron update that was blocked due to node-pty.
